Makefile

```
@@ -36,6 +36,8 @@ build-sda-sftp-inbox:
 	@cd sda-sftp-inbox && docker build -t ghcr.io/neicnordic/sensitive-data-archive:PR$$(date +%F)-sftp-inbox .
 build-sda-admin:
 	@cd sda-admin && go build
+build-sda-doa:
+	@cd sda-doa && docker build -t ghcr.io/neicnordic/sensitive-data-archive:PR$$(date +%F)-sda-doa .
```

```suggestion
	@cd sda-doa && docker build -t ghcr.io/neicnordic/sensitive-data-archive:PR$$(date +%F)-doa .
```

In order to match the rest of the artifacts generated.